docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -32,10 +32,7 @@ Prerequisites
 Software
 ~~~~~~~~
 
```

Comment:
Now the requirements for the development machine (Vagrant and VirtualBox) need to go here.